airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -94,3 +95,39 @@ def test_comments_not_found_ticket(prepare_stream_args, status_code, expected_co
                 next(comments)
         else:
             assert len(list(comments)) == expected_comment_count
+
+
+@pytest.mark.parametrize(
+    "input_data,expected_data",
+    [
+        # success
+        (
+            {"id": 123, "custom_fields": [{"id": 3213212, "value": ["fake_3000", "fake_5555"]}]},
+            {"id": 123, "custom_fields": [{"id": 3213212, "value": "['fake_3000', 'fake_5555']"}]},
+        ),
+        (
+            {"id": 234, "custom_fields": [{"id": 2345234, "value": "fake_123"}]},
+            {"id": 234, "custom_fields": [{"id": 2345234, "value": "fake_123"}]},
+        ),
+        (
+            {"id": 345, "custom_fields": [{"id": 5432123, "value": 55432.321}]},
+            {"id": 345, "custom_fields": [{"id": 5432123, "value": "55432.321"}]},
+        ),
+    ],
+)
+def test_transform_for_tickets_stream(prepare_stream_args, input_data, expected_data):
+    """Checks Transform in case when records come with invalid fields data types"""
+    stream = Tickets(**prepare_stream_args)
```

Comment:
What the point of testing it on stream level and manualy calling transorm method? Please test it on Source level like here connectors/source-iterable/unit_tests/test_source.py

Reply:
@avida I have not found a test in this file that would test Transformer.